ibm_vpd_utils.hpp

```diff
@@ -350,6 +351,28 @@ inline std::string createBindUnbindDriverCmnd(const std::string& devNameAddr,
             driverType + "/" + bindOrUnbind);
 }
 
+/**
+ * @brief Convert byte array to hex string.
+ * @param[in] vec - byte array
```

Comment:
its a templated argument. not just byte array. you can change the arg name..

Reply:
I'll change the description above and here, parameter should be fine, I think.
```suggestion
 * @param[in] vec - array
```